client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -345,6 +347,39 @@ export default React.createClass( {
 				section="domain-management" />;
 	},
 
+	pendingTransfer() {
+		const domains = this.getDomains().filter( domain => domain.pendingTransfer );
+
+		if ( domains.length !== 1 ) {
+			return null;
+		}
+
+		const domain = domains[ 0 ];
+
+		const compactNotice = this.translate( '%(domain)s is pending transfer.', {
+				args: {
+					domain: domain.name
+				}
+			} ),
+			fullNotice = this.translate( '%(domain)s is pending transfer. You must wait for the transfer to finish, and then update the settings at the new registrar.', {
```

Comment:
This line is way too long (don't make a "guess what's also too long" joke, don't make a "guess what's also too long" joke... I've been watching How I Met Your Mother too much lately, especially Barney Stinson).